project/Dependencies.scala

```diff
@@ -342,8 +343,13 @@ object Dependencies {
 
   val Sns = Seq(
     libraryDependencies ++= Seq(
-      "com.github.matsluni" %% "aws-spi-akka-http" % "0.0.4" excludeAll ExclusionRule(organization = "com.typesafe.akka"), // ApacheV2
-      "software.amazon.awssdk" % "sns" % AwsSdk2Version excludeAll(ExclusionRule(organization = "software.amazon.awssdk", name = "netty-nio-client"), ExclusionRule(organization = "io.netty")), // ApacheV2
+      "com.github.matsluni" %% "aws-spi-akka-http" % "0.0.4" excludeAll ExclusionRule(
+        organization = "com.typesafe.akka"
+      ), // ApacheV2
+      "software.amazon.awssdk" % "sns" % AwsSdk2Version excludeAll (ExclusionRule(
+        organization = "software.amazon.awssdk",
+        name = "netty-nio-client"
+      ), ExclusionRule(organization = "io.netty")), // ApacheV2
```

Comment:
Perhaps keep the wrapping the same, license on the similar relation to what's being licensed in these two, for convention reasons, since they are now different.

Reply:
Yea, scalafmt reformatted it this way...